src/components/Movie/Movie.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Actor from "../Actor/Actor";
+import "./style.scss";
+
+const Movie = ({ title, year, poster, rating, director, genre, cast }) => {
+  return (
+    <>
+      <article className="movie-card">
+        <p
+          className={
+            rating > 8.5
+              ? "movie-card__rating movie-card__rating--highlight"
+              : "movie-card__rating"
+          }
```

Comment:
když děláš styly pomocí css class, ještě by tě mohla zajímat knihovna https://www.npmjs.com/package/classnames, pak by se tohle dalo napsat jako:
```
className={classnames("movie-card__rating", {
  "movie-card__rating--highlight": rating > 8.5,
})}
```
tj ulehčuje to práci s podmíněnými třídami